doc/source/user/absolute_beginners.rst

```diff
@@ -1029,19 +1073,16 @@ argument. To find the unique rows, specify ``axis=0`` and for columns, specify
 
 To get the unique rows, index position, and occurrence count, you can use::
 
-  >>> unique_rows, indices, occurence_count = np.unique(a_2d, axis=0,
-  >>> return_counts=True, return_index=True)
-  >>> print('Unique Rows: ', '\n', unique_rows)
-  >>> print('Indices: ', '\n', indices)
-  >>> print('Occurrence Count:', '\n', occurence_count)
-  Unique Rows:
-    [[ 1  2  3  4]
-     [ 5  6  7  8]
-     [ 9 10 11 12]]
-  Indices:
-    [0 1 2]
-  Occurrence Count:
-    [2 1 1]
+  >>> unique_rows, indices, occurrence_count = np.unique(
+  ...      a_2d, axis=0, return_counts=True, return_index=True)
+  >>> print(unique_rows)
```

Comment:
Perhaps (and below)
```suggestion
  >>> unique_rows
```

Reply:
That changes the printing format: 
- adds `array(` before the text, 
- indents,
- adds commas between the values.

Reply:
Indeed - my suggestion is we should use that format

Reply:
+0, I'd leave that to Matti's discretion.  Presumably the original authors had a reason for using `print`, but when working in an interactive shell, I agree that  _not_  using `print` is more typical.  Then again, using a plain Python shell is not something I would recommend, and it would probably make sense to convert the whole thing to an ipython session, but that's out of scope for this PR.

Reply:
Agreed this is scope creep.

> Presumably the original authors had a reason for using print, 

The original author was not aware of the `>>>` convention for examples, and did the minimum work necessary to convert their print-based examples.